src/components/Table/index.tsx

```diff
@@ -77,6 +76,20 @@ const Table = <T,>({
 }: TableProps<T>): JSX.Element | null => {
   const totalCount = data.length;
 
+  const showSkeleton = useMemo(() => {
```

Comment:
It seems that useMemo might not be necessary in this case. There is no much calculation happening

Reply:
It's true that there is not much calculation in `showSkeleton` but `showSkeleton` is used by other fields that are memoized which otherwise would rerender all the time. You could argue that neither `showSkeleton` nor `tableData` or `tableColumns` are applying expensive transformations but I still think it should boost performance just a little. Or would you disagree?

Reply:
The performance impact is likely minimal either way, I don’t think it’s crucial to change it. I’m fine with keeping useMemo as it is